### PR TITLE
Develop

### LINKED ISSUE #503: Update press page
Update press page layout to resemble more of what the Blockstack Overview PDF press section's layout looks like.

- [x] Create SVG publication logos for press page on .org blockstack/designs#618
- [x] Update press page to build out new component and add articles from https://blockstack.com/blockstack_overview.pdf to the press page
- [ ] Press - Add one of the articles about the token sale

----

screen shot 2018-02-09 at 2 07 48 pm

### PR DESCRIPTION
**Pull request by guylepage3**
Closes #503 Update press page

### CODE REVIEW

**Comment by guylepage3** [comment]
@yknl @larrysalibra can either of you check this out and merge when you get a moment? thank you. 👍

**Comment by larrysalibra** [comment]
Sure thing @guylepage3 !

File: package-lock.json [hunk 1]
```diff
@@ -6110,7 +6110,11 @@
           "integrity": "sha512-Z5BTk6ZRe4tXXQlkqftmsAUANpXmuwlsF5Oov8ThoMbQRzdGTA1ngYRW160GexgOgjsFOKJz0LYhoNi+2AMBUw==",
           "dev": true,
           "requires": {
+<<<<<<< HEAD
```
Comment (larrysalibra) [inline package-lock.json]: looks like `package-lock.json` has some unmerged changes that are causing the netlify deploy preview to fail.

This should be fixable by doing the following:
```
rm package-lock.json
npm install
git add package-lock.json
git commit -S 
```

Comment (guylepage3) [reply package-lock.json]: complete. no issues.

**Comment by guylepage3** [comment]
@larrysalibra this should be ready to go now.

**Comment by larrysalibra** [comment]
Works well for me now! Looks great! Awesome work @guylepage3 and @cwackerfuss

**Pull request merged**